forte/results.py

```diff
@@ -0,0 +1,20 @@
+class Results:
+    """
+    A class used to store the results of a computation.
+    """
+    def __init__(self):
+        self._data = dict()
+
+    def add(self, label, value, description, units):
```

Comment:
Is there a way for a user to access a description?

Reply:
Added access to description and units.